site/src/docs/configuration/reproxy/index.md

```diff
@@ -1,10 +1,13 @@
 ---
 title: Reproxy
+menuTitle: Reproxy
```

Comment:
it's unnecessary to add `menuTitle` if you don't want to redefine `title`. By default `title` is used as the menu title.